ci/travis/determine_tests_to_run.py

```diff
@@ -0,0 +1,105 @@
+# Script used for checking changes for incremental testing cases
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import subprocess
+
+
+def list_changed_files(commit_range):
+    """Returns the list of files changed within the specified commit range. If
+    an error occurs while running git, a RuntimeError will be raised.
```

Comment:
Please follow the doc string style described in https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Something like

```python
"""Brief description that fits in one line.

More information about the function, which can
take multiple lines.
```